main.tf

```
@@ -49,7 +49,8 @@ data "aws_iam_policy_document" "lambda" {
     actions = [
       "organizations:DetachPolicy",
       "organizations:AttachPolicy",
-      "organizations:ListPolicies"
+      "organizations:ListPolicies",
+      "organizations:ListPoliciesForTarget"
```

Just to minimize future diffs, let's add the trailing comma...
```suggestion
      "organizations:ListPoliciesForTarget",
```